manager/manager.go

```diff
@@ -1237,6 +1237,24 @@ func (m *manager) watchForNewOoms() error {
 			if err != nil {
 				klog.Errorf("failed to add OOM kill event for %q: %v", oomInstance.ContainerName, err)
 			}
+
+			// Count OOM events for later collection by prometheus
+			request := v2.RequestOptions{
+				IdType: v2.TypeName,
+				Count:  1,
+			}
+			conts, err := m.getRequestedContainers(oomInstance.ContainerName, request)
+			if err != nil {
+				klog.Errorf("failed getting container info for %q: %v", oomInstance.ContainerName, err)
```

Comment:
This is not an error if it's fine to `continue`.

Reply:
Sure, what's the wanted log level? (`Warningf`?)

Reply:
`klog.V(2).Info()` could make some sense, I think.